server/sonar-server/src/main/java/org/sonar/server/user/ws/CurrentAction.java

```diff
@@ -95,6 +98,7 @@ private CurrentWsResponse toWsResponse(DbSession dbSession, String userLogin) {
       .setPermissions(Permissions.newBuilder().addAllGlobal(getGlobalPermissions()).build())
       .setShowOnboardingTutorial(!user.isOnboarded());
     setNullable(emptyToNull(user.getEmail()), builder::setEmail);
```

Comment:
Existing email fields should be deprecated => Please update the JIRA ticket to say 'email' field is deprecated and add a changelog entry (line 65)